src/components/Forms/validators.js

```diff
@@ -0,0 +1,19 @@
+// validation functions to use
+
+export const validateNumber = (number) => {
+  const rgxp = /[(0-9)+.?(0-9)*]+/gim;
```

Comment:
Here I am not sure about moder JS interpeter optimisations. And after all this place will never be a hardest bottleneck.
But probably it is better to move regular expression definitions out of function scope to the file scope, to avoid creating a new regexp on every function call.

Reply:
solved https://github.com/maxSedelnikov/react-shopping-app/pull/1/commits/24984c3c8b52e6eb5707ea4a4edfa8373dc5a402